pkg/operator/ceph/cluster/osd/status.go

```diff
@@ -197,7 +197,7 @@ func (c *Cluster) completeOSDsForAllNodes(config *provisionConfig, configOSDs bo
 					w.Stop()
 					<-time.After(5 * time.Second)
 					leftNodes := 0
-					leftRemaingNodes := util.NewSet()
```

Comment:
This change is unnecessary.

Reply:
See above (I'll remove this change as well for now but I'll bring it back in another PR).